Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java

```diff
@@ -100,21 +96,6 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
       mBinding.menuRefreshWiimotes.setVisibility(View.GONE);
     }
 
-    int bottomPaddingRequired = getBottomPaddingRequired();
-
-    // Provide a safe zone between the navigation bar and Exit Emulation to avoid accidental touches
-    float density = getResources().getDisplayMetrics().density;
-    if (bottomPaddingRequired >= 32 * density)
-    {
-      bottomPaddingRequired += 32 * density;
-    }
-
-    if (bottomPaddingRequired > view.getPaddingBottom())
-    {
-      view.setPadding(view.getPaddingLeft(), view.getPaddingTop(),
-              view.getPaddingRight(), bottomPaddingRequired);
-    }
```

Comment:
This code existed for a reason.

Imagine you're in portrait mode and you have button navigation enabled. To open the menu, you need to swipe from the bottom of the screen to open the navigation bar, and then you need to press Back in the navigation bar. With this code removed, the Exit Emulation button shows up right *under* the Back button, meaning you have to wait for the navigation bar to disappear before you can actually press Exit Emulation. (Unless you want to try to hit the tiny part of the button that isn't obscured, but that's unreasonable.)